src/features/movies/moviesSlice.js

```diff
@@ -0,0 +1,41 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const moviesSlice = createSlice({
+  name: "movies",
+  initialState: {
+    movies: [],
+    genres: [],
+    loadingStatus: "loading",
+  },
+  reducers: {
+    setMovies: (state, { payload: movies }) => {
+      state.movies = movies;
+    },
+    setGenres: (state, { payload: genres }) => {
+      state.genres = genres;
+    },
+    setLoadingToSucces: (state) => {
+      state.loadingStatus = "success";
+    },
+    setLoadingToFail: (state) => {
+      state.loadingStatus = "fail";
+    },
```

Comment:
Same here. If we call it smth like fetchMoviesSuccess and fetchMoviesError we're reffering to fetching process